mssql/env.example.sh

```diff
@@ -0,0 +1,7 @@
+CONTAINER_NAME="mssql"
+MSSQL_SA_PASSWORD="x3h3Ays7dNdUGkYVKyDEMkHVEUeFyDr8"
+MSSQL_PID="Enterprise"
```

Comment:
Add readme to `mssql` folder, and explain which values support this field.

Reply:
you can reach what it is for from this link https://learn.microsoft.com/en-us/sql/linux/sql-server-linux-configure-environment-variables?view=sql-server-ver16